doc/howto/network_increase_bandwidth.md

```diff
@@ -1,45 +1,44 @@
-interface `txqueuelen` to 10000.
+```{note}
+The following instructions use a `txqueuelen` value of 10000, which is commonly used with 10GbE NICs, and a `net.core.netdev_max_backlog` value of 182757.
+Depending on your network, you might need to use different values.
 
-In Debian-based distributions, you can change `txqueuelen` permanently in `/etc/network/interfaces`.
-You can add for example: `up ip link set eth0 txqueuelen 10000` to your interface configuration to set the `txqueuelen` value on boot.
-You could set `txqueuelen` temporary (for test purpose) with `ifconfig <interface> txqueuelen 10000`.
+In general, you should use small `txqueuelen` values with slow devices with a high latency, and high `txqueuelen` values with devices with a low latency.
+For the `net.core.netdev_max_backlog` value, a good guideline is to use the minimum value of the `net.ipv4.tcp_mem` configuration.
+```
 
-##### `/etc/sysctl.conf`
+## Increase the network bandwidth on the LXD host
 
-You also need to increase `net.core.netdev_max_backlog` value.
-You can add `net.core.netdev_max_backlog = 182757` to `/etc/sysctl.conf` to set it permanently (after reboot)
-You set `netdev_max_backlog` temporary (for test purpose) with `echo 182757 > /proc/sys/net/core/netdev_max_backlog`
-Note: You can find this value too high, most people prefer set `netdev_max_backlog` = `net.ipv4.tcp_mem` min. value.
-For example I use this values `net.ipv4.tcp_mem = 182757 243679 365514`
+Complete the following steps to increase the network bandwidth on the LXD host:
 
-#### Containers changes
+1. Increase the transmit queue length (`txqueuelen`) of both the real NIC and the LXD NIC (for example, `lxdbr0`).
+   You can do this temporarily for testing with the following command:
 
-You also need to change the `txqueuelen` value for all your Ethernet interfaces in containers.
-In Debian-based distributions, you can change `txqueuelen` permanently in `/etc/network/interfaces`.
-You can add for example `up ip link set eth0 txqueuelen 10000` to your interface configuration to set the `txqueuelen` value on boot.
+       ifconfig <interface> txqueuelen 10000
 
-#### Notes regarding this change
+   To make the change permanent, add the following command to your interface configuration in `/etc/network/interfaces`:
 
-10000 `txqueuelen` value commonly used with 10GbE NICs. Basically small
-`txqueuelen` values used with slow devices with a high latency, and higher
-with devices with low latency. I personally have like 3-5% improvement
-with these settings for local (host with container, container vs
-container) and internet connections. Good thing about `txqueuelen` value
-tweak, the more containers you use, the more you can be can benefit from
-this tweak. And you can always temporary set this values and check this
-tweak in your environment without LXD host reboot.
+       up ip link set eth0 txqueuelen 10000
+
+1. Increase the receive queue length (`net.core.netdev_max_backlog`).
```

Comment:
Shouldn't this be `2.`?

Reply:
Markdown allows using `1.` for all steps (and will render them correctly numbered). The advantage of that is that if you add or move a step, the numbering in the output will still be correct.

Reply:
Cool, I didn't know about that.